packages/teams-js/src/public/search.ts

```diff
@@ -127,4 +128,14 @@ export namespace search {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.search ? true : false;
   }
+
+  /**
+   * Clear the host M365 application's search box
+   *
+   * @beta
+   */
+  export function closeSearch(): void {
```

Comment:
Have we finalized on the name? Is it going to be closeSearch or clearSearch?

Reply:
I do have to agree with Trevor's comment on the matter I think we first need to define exactly what we want with the method if it is just to clear the text that appears on the search box or to exit completely from the experience